ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=syncd container
-Requires=database.service
-After=database.service
+Requires=database.service swss.service
+After=database.service swss.service
```

Comment:
syncd is not depending on swss
and syncd should start before swss
@vitaliy-senchyshyn @lguohan